tests/phpunit/tests/Checker/Checks/Plugin_Readme_Check_Tests.php

```diff
@@ -67,8 +67,6 @@ public function test_run_with_errors_empty_name() {
 
 		$this->assertNotEmpty( $errors );
 		$this->assertArrayHasKey( 'readme.txt', $errors );
-		// Two errors are 'empty_plugin_name' and 'no_license'.
```

Comment:
@swissspidy I have removed this to fix the PHPUnit error. We are gradually adding new readme checks in the plugin. Every time we add new checks this number does not match. Since our target here is to check for empty plugin name which is below this code, test should be good for this I guess.